repose-aggregator/services/service-client/api/src/main/java/org/openrepose/core/services/serviceclient/akka/AkkaServiceClient.java

```diff
@@ -29,6 +29,8 @@
  */
 public interface AkkaServiceClient {
 
+    void setConnectionPoolId(String connectionPoolId);
```

Comment:
We can give the factory a destroy method too, if we want to make sure we get clean teardowns of the actor system